### PR TITLE
Improve interface

### PR DESCRIPTION
**Pull request by thinhbd-1256**
Improve index, login and signup UI

### CODE REVIEW

**Comment by thinhbd-1256** [comment]
ready

File: app/views/events/_event.html.erb [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-    <div class="detailImg">
-      <img src="/assets/event3.jpg" height="100%" >
-      <div class="time">
-        <p><%= event.date_start %></p>
-        <p><%= event.date_end %></p>
+<div class = "row">
+  <div class="col-lg-6 col-lg-offset-3">
+    <div class="detailThumbnail right">
+      <div class="detailImg">
+        <img src="/assets/event3.jpg" height="100%" >
```
Comment (huongnguyenmta) [inline app/views/events/_event.html.erb]: move vào file css nhé: `height="100%"`

File: app/views/events/_event.html.erb [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-      <p class="address"><%= event.content %></p>
-      <p class="create">By <span><%= event.user.name %></span><span class="category"></span></p>
-      <div class = "button-event clearfix pull-right">
-        <%= link_to t(".interest"), "#", class: "btn btn-primary btn-event" %>
-        <%= link_to t(".join"), "#", class: "btn btn-primary btn-event" %>
+      <div class="detailDes">
+        <p class="title"><%= link_to event.title, event %></p>
+        <p class="address"><%= event.content %></p>
+        <p class="create"><%= t ".by" %> <span><%= event.user.name %></span><span class="category"></span></p>
+        <div class = "button-event clearfix pull-right">
```
Comment (huongnguyenmta) [inline app/views/events/_event.html.erb]: #good
`class="button-event clearfix pull-right"`

File: app/views/layouts/_rails_default.html.erb [hunk 3]
```diff
@@ -1,3 +1,7 @@
 <%= csrf_meta_tags %>
-<%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
+<%= stylesheet_link_tag "application",
+ "https://fonts.googleapis.com/css?family=Bungee+Outline",
+ "https://fonts.googleapis.com/css?family=Montserrat:900",
+ "https://fonts.googleapis.com/css?family=Montserrat+Alternates",
+ media: "all", "data-turbolinks-track": "reload" %>
```
Comment (huongnguyenmta) [inline app/views/layouts/_rails_default.html.erb]: indent từ dòng 3-6

File: app/views/sessions/new.html.erb [hunk 4]
```diff
@@ -1,22 +1,23 @@
 <h1 class = "text-center"><%= t ".tit" %></h1>
 <div class="row login-form">
-  <div class="col-md-6 col-lg-6 col-lg-offset-3 col-md-offset-3">
+  <div class="col-lg-4 col-lg-offset-4 col-md-4 col-md-offset-4">
+    <h1 class = "form-title">Login</h1>
```
Comment (huongnguyenmta) [inline app/views/sessions/new.html.erb]: I18n

**Comment by huongnguyenmta** [comment]
commented

**Comment by thinhbd-1256** [comment]
ready

**Comment by hungnh103** [comment]
(merged)
:D

**Pull request merged**